src/pages/dar_application/DataUseAgreements_new.js

```diff
@@ -0,0 +1,82 @@
+
+import { a, div, h2, h3, span, li, ol, p} from 'react-hyperscript-helpers';
+import {isNil} from 'lodash/fp';
+import BroadLibraryCardAgreementLink from '../../assets/Library_Card_Agreement_2021.pdf';
+import NihLibraryCardAgreementLink from '../../assets/NIH_Library_Card_Agreement_11_17_22_version.pdf';
+
+import './dar_application_new.css';
+
+export default function DataUseAgreements(props) {
+
+  const {
+    save,
+    attestAndSend,
+    darCode,
+  } = props;
+
+  return (
+    div({
+      className: 'dar-step-card'
+    }, [
+
+      h2({}, ['Step 4: Attestation and Agreement(s)']),
```

Comment:
This is slightly different from the mockup which has "Data Use Agreements" for the header